dev/gpctl/cmd/public-api-workspaces-get.go

```diff
@@ -6,7 +6,8 @@ package cmd
 
 import (
 	"github.com/gitpod-io/gitpod/common-go/log"
-	v1 "github.com/gitpod-io/gitpod/components/public-api/go/experimental/v1"
+	v1 "github.com/gitpod-io/gitpod/components/public-api/go/v1"
```

Comment:
@iQQBot could you approve it?

@easyCZ Would it be alright if ownership of gpctl or public-api-* files are shared with EXP team?